test/src/test/java/hudson/model/UsageStatisticsTest.java

```diff
@@ -40,11 +44,16 @@
 /**
  * @author Kohsuke Kawaguchi
  */
-public class UsageStatisticsTest extends HudsonTestCase {
+public class UsageStatisticsTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
Unfortunately the JenkinsRule is required even if not obvious at first sight.

``UsageStatistics`` accesses the Jenkins instance singleton internally.